src/main/kotlin/com/sourcegraph/cody/history/HistoryService.kt

```diff
@@ -9,11 +9,7 @@ import com.intellij.openapi.project.Project
 import com.sourcegraph.cody.agent.protocol.ChatMessage
 import com.sourcegraph.cody.agent.protocol.Speaker
 import com.sourcegraph.cody.config.CodyAuthenticationManager
-import com.sourcegraph.cody.history.state.ChatState
-import com.sourcegraph.cody.history.state.EnhancedContextState
-import com.sourcegraph.cody.history.state.HistoryState
-import com.sourcegraph.cody.history.state.LLMState
-import com.sourcegraph.cody.history.state.MessageState
+import com.sourcegraph.cody.history.state.*
```

Comment:
Personally I do not like wildcard imports (and [here](https://stackoverflow.com/a/147461) is why). 

You can change your IDE settings (and [here](https://stackoverflow.com/a/3348855) is how).

Reply:
Done